org.d3cod3.Mosaic.yml

```diff
@@ -0,0 +1,431 @@
+    sources:
+    - type: archive
+      url: https://github.com/pocoproject/poco/archive/poco-1.10.1-release.tar.gz
+      sha256: 44592a488d2830c0b4f3bfe4ae41f0c46abbfad49828d938714444e858a00818
+  - name: libdaemon
+    buildsystem: autotools
+    config-opts:
+      - "--prefix=/app"
+    sources:
+      - type: git
+        url: git://git.0pointer.net/libdaemon.git
+        commit: 9fcc28e0e8f84968d1fb8b6d544a42efb13803ec
+  - name: libevent
+    buildsystem: autotools
+    config-opts:
+      - "--prefix=/app"
+    sources:
+      - type: git
+        url: https://github.com/libevent/libevent
+        tag: release-2.1.12-stable
+  - name: avahi
+    buildsystem: autotools
+    config-opts:
+      - "--prefix=/app"
+      - "--with-distro=none"
+      - "--disable-gtk3"
+      - "--disable-mono"
+      - "--disable-qt5"
+      - "--disable-manpages"
+      - "--disable-python"
+      - "--with-systemdsystemunitdir=no"
+    sources:
+      - type: git
+        url: https://github.com/lathiat/avahi
+        tag: v0.8
+  - name: openframeworks
+    buildsystem: simple
+    # subdir: libs/openFrameworksCompiled/project
+    build-commands:
+      - cd libs/openFrameworksCompiled/project && make -j$FLATPAK_BUILDER_N_JOBS Debug
+      - cd src/fftw3 && ./configure --prefix=`pwd` --enable-float --enable-sse2 --with-incoming-stack-boundary=2 --with-our-malloc16 --disable-shared --enable-static && make MAKEINFO=true -j$FLATPAK_BUILDER_N_JOBS
+      - mkdir addons/ofxAudioAnalyzer/libs/fftw3f/lib/linux64
+      - cp src/fftw3/.libs/libfftw3f.a addons/ofxAudioAnalyzer/libs/fftw3f/lib/linux64/
+      - cp addons/ofxNDI/libs/libndi/lib/x86_64-linux-gnu/libndi.so.3.7.1 /app/lib
+      - ln -s /app/lib/libndi.so.3.7.1 /app/lib/libndi.so.3
+      - cd apps/d3cod3/Mosaic && make -j$FLATPAK_BUILDER_N_JOBS Release
+      - mkdir /app/mosaic
+      - cp -ar * /app/mosaic
+    post-install:
+      - ln -s ../mosaic/apps/d3cod3/Mosaic/bin/Mosaic /app/bin/mosaic
+    cleanup:
+      - src
+    sources:
+# openframeworks
+      - type: archive
+        url: https://openframeworks.cc/versions/v0.11.0/of_v0.11.0_linux64gcc6_release.tar.gz
+        sha256: 513cee3ce081e79fdc1a942af09869145fff3d7addeee0a76cef26fbc3ae69b1
+      - type: shell
+        commands: 
+          - sed -e '/GL\/glext.h/ s/^#*/\/\//' -i libs/openFrameworks/utils/ofConstants.h
+# use openCV 4
+      - type: shell
+        commands:
+          - sed -i -E 's/ADDON_PKG_CONFIG_LIBRARIES =(.*)opencv\s/ADDON_PKG_CONFIG_LIBRARIES =\1opencv4 /g' addons/ofxOpenCv/addon_config.mk
+          - sed -i -E 's/ADDON_PKG_CONFIG_LIBRARIES =(.*)opencv$/ADDON_PKG_CONFIG_LIBRARIES =\1opencv4/g' addons/ofxOpenCv/addon_config.mk
+      - type: patch
+        path: openal_fix.patch
+        strip-components: 0
+      - type: patch
+        path: make4.3.patch
+        strip-components: 0
+# addons
+      - type: git
+        url: https://github.com/d3cod3/ofxAudioAnalyzer.git
+        branch: master
+        # tag:
```

Comment:
Should be a tagged release or a specific commit, the same for other addons?

Reply:
Hmmm, there is a commit though. Those addons are not tagged yet I fear